Configuration/HLT/python/autoCondHLT.py

```diff
@@ -1,5 +1,7 @@
 # https://cms-conddb.cern.ch/browser/#search
 
+snapshotTime = "9999-12-31 23:59:59.000"
```

Comment:
OK, here is the long version of the summary...

The best approach would be to use an independent timestamp for each tag.
When we update or add a new customisation, we use the current¹ time for the relevant tag, and leave unchanged the timestamps for the other tags.

As an intermediate measure, we use the  current¹ time for all tags. If you are OK with migrating to independent time stamps, even better.

.A


¹ current as in "at the time the pull request is made"